### PR TITLE
remove ergo and paper options in create/restore

### PR DESCRIPTION
**Pull request by zuzunker**
https://emurgo.atlassian.net/browse/YOEXT-633

### CODE REVIEW

**Comment by neuodev** [comment]
[autoscript] ⚠️ `ruslan/remove-ergo-and-paper-options` doesn't full our branch naming convention.
Should follow `<tag>/<jira-ticket-id>/<short-name>`
<details>
<summary>Details</summary>

- `<tag>`: `fix`, `feat`, `chore`, `refactor`, `test`, ...
- `<jira-ticket-id>`: The issue ID of JIRA all capital letters
- `<short-name>`: A short and descriptive name for the branch
</details>

[](__JIRA_COMMENT_FLAG)

**Review by zuzunker — COMMENTED** [review]
/check

File: packages/yoroi-extension/features/step_definitions/wallet-creation-steps.js [hunk 1]
```diff
@@ -39,13 +36,8 @@ When(/^I select the currency ([^"]*)$/, async function (currency) {
...
-When(/^I select Create Wallet$/, async function () {
-  await this.waitForElement(createOptionDialog);
-  await this.click(createNormalWalletButton);
-});
-When(/^I select Paper Wallet$/, async function () {
-  await this.waitForElement(createOptionDialog);
-  await this.click(createPaperWalletButton);
+When(/^I select Create Wallet$/, () => {
+  // <TODO:PENDING_REMOVAL>
 });
```
Comment (neuodev) [inline packages/yoroi-extension/features/step_definitions/wallet-creation-steps.js]: Empty test block! Is done on purpose?
```suggestion
```

Comment (Nebyt) [reply packages/yoroi-extension/features/step_definitions/wallet-creation-steps.js]: Looks like I forget to delete it :face_in_clouds:

**Review by Nebyt — CHANGES_REQUESTED** [review]
@vsubhuman Please rollback changes from the folder `../packages/yoroi-extension/features`. I will remove outdated info myself.

**Review by zuzunker — COMMENTED** [review]
@Nebyt , `/features` content is rolled back, /check now plz

**Review by Nebyt — APPROVED** [review]

**Pull request merged**